src/main/java/com/ibm/watson/apis/conversation_with_discovery/utils/Constants.java

```diff
@@ -26,7 +27,7 @@ private Constants() {
   public static final String CONVERSATION_URL = "https://gateway.watsonplatform.net/conversation/api";
   
   /** The Constant CONVERSATION_VERSION. */
-  public static final String CONVERSATION_VERSION = ConversationService.VERSION_DATE_2016_09_20;
+  public static final String CONVERSATION_VERSION = Conversation.VERSION_DATE_2017_05_26;
```

Comment:
Let's use Strings for our default values. We are going to remove the constants from our SDkKs